sql/core/src/main/scala/org/apache/spark/sql/execution/arrow/ArrowUtils.scala

```diff
@@ -31,7 +31,8 @@ object ArrowUtils {
 
   // todo: support more types.
 
-  def toArrowType(dt: DataType): ArrowType = dt match {
+  /** Maps data type from Spark to Arrow. NOTE: timeZoneId required for TimestampTypes */
+  def toArrowType(dt: DataType, timeZoneId: Option[String]): ArrowType = dt match {
```

Comment:
Why do we use `Option` if this is not allowed to be `None`?

Reply:
Hmm, I see, it's required only if dt is `TimestampType`.

Reply:
Btw, why don't we set default values for `timeZoneId`? Seems a bit weird having to specify `None` for other types.

Reply:
I don't think it's a good idea to have a default value for `timeZoneId`, then it makes it very easy to forget it and miss places in the code that really should have a timezone.  It will compile and probably pass tests, but might fail somewhere down the line.  Any place in Spark should specify a timezone, so it's better to force an argument to be used.  Only in the tests, where we are sure no TimestampTypes are used would we set it to None.

Reply:
Ok. Makes sense. If this arg is required. We should maybe make it just `String` instead of `Option[String]`, I find having a required arg to be `Option` is not intuitive..

Reply:
Btw, this is a bit nit pick. So feel free to stick to this if you think `Option[String]` is the right thing.